vmdb/app/views/shared/dialogs/_dialog_field_drop_down_list.haml

```diff
@@ -1,10 +1,6 @@
 - if edit
-  - if field.values.length > 1
-    - select_values = field.values.collect(&:reverse)
-    = select_tag(field.name, options_for_select(select_values, field.default_value), drop_down_options(field, url))
-
-  - else
-    = h(field.values[0].last) if !field.values.empty?
+  - select_values = field.values.collect(&:reverse)
+  = select_tag(field.name, options_for_select(select_values, field.default_value), drop_down_options(field, url))
```

Comment:
Did I read this correctly:

Old:

count|code
---|---
0|display nothing
1|display value
2|display select

New:

show a select (possibly empty)

goal being we can then populate it in the future